integration-tests/resources/wallets/transfer_from_pos_vault.rho

```diff
@@ -0,0 +1,27 @@
+new return, rl(`rho:registry:lookup`), poSCh, stdout(`rho:io:stdout`), 
+     deployerId(`rho:rchain:deployerId`), transferRetCh, log in {
+   rl!(`rho:rchain:pos`, *poSCh) |
+   stdout!(("start rholang code")) |
+   match (
+    "%TARGET_ADDR",
+    %AMOUNT
+   ) {
+       (targetAddr, amount) => {
+         stdout!(("start contract")) |
+         for(@(_, PoS) <- poSCh) {
+           @PoS!("posVaultTransfer", targetAddr, amount, *deployerId, *transferRetCh) |
+           stdout!(("contract executed")) | 
+           for (@result <- transferRetCh){
+             match result{
+               (true, _) =>{log!("%LOG_MARKER Successfully reason: ${reason}" %%{"reason": "Nil"})}
+               (false, reason) => {log!("%LOG_MARKER Failing reason: ${reason}" %%{"reason": reason}) | stdout!("2. contract executed")}
+            }
+          }   
+        } 
+      }
+    } | 
+    
+    contract log(@data) = {
```

Comment:
Sorry that the example is including this `log` stuff. The log is actually not needed. You could remove this `log` contract and replase `log!(xxxx)` with `stdout!(xxxx)`